src/assets/products.json

```diff
@@ -7,6 +7,10 @@
       "size": 1.0,
       "isPerishable": true,
       "perishable": 15
+    },
+    "probability": {
```

Comment:
I think `demand` makes more sense than `probability`. I understand these are parameters to a probability distribution, but their function is directly related to the demand, not some abstract concept `probability`.